pavelib/quality.py

```diff
@@ -258,6 +264,16 @@ def run_quality(options):
     pep8_reports = u' '.join(pep8_files)
 
     try:
+        violations_str = None
+        with open(pep8_reports, 'r') as f:
+            violations_list = f.readlines()
+            violations_str = '\n'.join(violations_list)
+
+        if violations_str:
+            raise BuildFailure(
```

Comment:
Yah, this was just for testing because `print` wasn't printing to console directly. I wanted to make sure that the `violations_str` was being build up appropriately.